esmvaltool/utils/batch-jobs/job_DKRZ-EG.sh

```diff
@@ -3,29 +3,26 @@
 ### BATCH SCRIPT TO RUN THE ESMVALTOOL AT DKRZ MISTRAL
```

Comment:
I think this file should not be included in the PR, but I appreciate that it helps me test the recipe! :)

Reply:
Hey @bsolino, thanks for looking at my PR. Haha, I do my best to help :sweat_smile: 
And I will try to look at those issues soon. 
Cheers,